Doc/library/ast.rst

```diff
@@ -402,15 +377,12 @@ Expressions
 
    .. doctest::
 
-        >>> print(ast.dump(ast.parse("x + y"), indent=4))
-        Module(
-            body=[
-                Expr(
-                    value=BinOp(
-                        left=Name(id='x', ctx=Load()),
-                        op=Add(),
-                        right=Name(id='y', ctx=Load())))],
-            type_ignores=[])
+        >>> print(ast.dump(ast.parse("x + y",mode="eval"), indent=4))
```

Comment:
Please always add a space after comma.

It would be nice to use single quotes which are default in Python.
```suggestion
        >>> print(ast.dump(ast.parse('x + y', mode='eval'), indent=4))
```

Reply:
:+1:  I will make sure all the examples follow this